extensions/ql-vscode/src/vscode-tests/no-workspace/data/astBuilder.json

```diff
@@ -18,7 +18,7 @@
       [
         {
           "id": 26359,
-          "label": "",
+          "label": "SHOULD NOT USE ",
```

Comment:
I like having a fixed test like this, but we have to keep it up to date. Is it feasible to have an integration test that runs a real AST query and checks we can handle the results?

Reply:
We can do that. I think we should create a new issue that lists a bunch of integration tests that we should add once the integration test PR is merged. I'll do that right now.

Reply:
#676 is the start. Feel free to add more.